app/models/requirement.rb

```diff
@@ -2,14 +2,12 @@ class Requirement < ApplicationRecord
   has_many :requirement_block_requirements, dependent: :destroy
   has_many :requirement_blocks, through: :requirement_block_requirements
 
-  enum input_type: { text: 0, number: 1, checkbox: 2, select: 3, multi_option_select: 4, date: 5 }, _prefix: true
+  enum input_type: { textfield: 0, number: 1, checkbox: 2, select: 3, multi_option_select: 4, date: 5 }, _prefix: true
```

Comment:
any reason to change it to `textfield`? Seems a bit of a mismatch with the other types as they do not have field in their names.